injective/assetlist.json

```diff
@@ -123,6 +123,31 @@
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/injective/images/autism.png"
         }
       ]
+    },
+    {
+      "description": "The $WGMI Token - We Gonna Make It. Are you ready?",
+      "denom_units": [
+        {
+          "denom": "factory/inj1rmjzj9fn47kdmfk4f3z39qr6czexxe0yjyc546/WGMI",
+          "exponent": 0
+        },
+        {
+          "denom": "WGMI",
+          "exponent": 6
+        }
+      ],
+      "base": "factory/inj1rmjzj9fn47kdmfk4f3z39qr6czexxe0yjyc546/WGMI",
+      "name": "The $WGMI Token - We Gonna Make It. Are you ready?",
```

Comment:
Can you please save this for the description and use a much shorter name for the token?

Reply:
fixed